be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.h

```diff
@@ -49,6 +50,7 @@ class AggregateBlockingSinkOperator : public Operator {
     [[nodiscard]] Status reset_state(RuntimeState* state, const std::vector<ChunkPtr>& refill_chunks) override;
 
 protected:
+    DECLARE_ONCE_DETECTOR(_set_finishing_once);
```

Comment:
move this variable to Operator?

Reply:
Some operators are inherited relationships, so they are not suitable for moving into operators. such as spillablexxxoperators